dotcom-rendering/src/components/MiniProfile.tsx

```diff
@@ -0,0 +1,186 @@
+	a:not(:hover) {
+		text-decoration-color: ${neutral[86]};
+	}
+	a:hover {
+		text-decoration: underline;
+	}
+	ul {
+		list-style: none;
+		margin: 0 0 0.75rem;
+		padding: 0;
+		margin-bottom: 0.5rem;
+	}
+	ul li {
+		padding-left: 1.25rem;
+	}
+	ul li p {
+		display: inline-block;
+		margin-bottom: 0;
+	}
+	ul li:before {
+		display: inline-block;
+		content: '';
+		border-radius: 0.375rem;
+		height: 10px;
+		width: 10px;
+		margin: 0 0.5rem 0 -1.25rem;
+		background-color: ${palette('--bullet-fill')};
+	}
+	strong {
+		font-weight: bold;
+	}
+`;
+
+const endNoteStyles = css`
+	${textSans14};
+	line-height: 135%;
+	color: ${palette('--mini-profiles-text-subdued')};
+	margin-bottom: 1rem;
+`;
+
+const bottomBorderStyles = css`
+	border-top: 1px solid ${palette('--article-border')};
+	margin-bottom: 0.5rem;
+`;
+
+const headingMarginStyle = css`
+	margin-bottom: 4px;
+`;
+
+interface MiniProfileProps {
+	format: ArticleFormat;
+	ajaxUrl: string;
+	host?: string;
+	pageId: string;
+	isAdFreeUser: boolean;
+	isSensitive: boolean;
+	abTests: ServerSideTests;
+	switches: Switches;
+	editionId: EditionId;
+	hideCaption?: boolean;
+	starRating?: StarRating;
+	miniProfile: MiniProfileModel;
+	RenderArticleElement: ArticleElementRenderer;
+}
+
+export const MiniProfile = ({
+	miniProfile,
+	format,
+	ajaxUrl,
+	host,
+	pageId,
+	isAdFreeUser,
+	isSensitive,
+	switches,
+	abTests,
+	editionId,
+	hideCaption,
+	starRating,
+	RenderArticleElement,
+}: MiniProfileProps) => {
+	return (
+		<>
+			<li css={miniProfileStyles} data-spacefinder-role="nested">
+				<hr css={headingLineStyles} />
+				<h3
+					id={slugify(miniProfile.title)}
+					css={[subheadingStyles(format), headingMarginStyle]}
+				>
+					{miniProfile.title}
+				</h3>
+				<Bio html={miniProfile.bio} />
+				{miniProfile.body.map((element, index) => (
+					<RenderArticleElement
+						// eslint-disable-next-line react/no-array-index-key -- This is only rendered once so we can safely use index to suppress the warning
+						key={index}
+						format={format}
+						element={element}
+						ajaxUrl={ajaxUrl}
+						host={host}
+						index={index}
+						isMainMedia={false}
+						pageId={pageId}
+						webTitle={miniProfile.title}
+						isAdFreeUser={isAdFreeUser}
+						isSensitive={isSensitive}
+						switches={switches}
+						abTests={abTests}
+						editionId={editionId}
+						hideCaption={hideCaption}
+						starRating={starRating}
+						forceDropCap="off"
+						isListElement={true}
+					/>
+				))}
+				{miniProfile.endNote ? (
+					<EndNote text={miniProfile.endNote} />
+				) : null}
+			</li>
+		</>
+	);
+};
+
+const Bio = ({ html }: { html?: string }) => {
+	if (!html) return null;
+	const sanitizedHtml = sanitiseHTML(html, {});
+	return (
+		<>
+			<div
+				css={bioStyles}
+				dangerouslySetInnerHTML={{ __html: sanitizedHtml }}
+			/>
+			<div css={bottomBorderStyles} />
+		</>
+	);
+};
+
+const EndNote = ({ text }: { text?: string }) => {
+	if (!text) return null;
```

Comment:
What about the empty string, i.e. `text.trim() === ""`?

Reply:
In this case I don't think we want to render anything if there is no text provided, whether that's `undefined` or `""`